views/includes/page-templates/trust-involved.html

```diff
@@ -212,7 +212,7 @@ <h1 class="govuk-heading-m">
         } %}
         {% include "includes/inputs/radio-input.html" %}
 
-        {% if pageData.isRelevantPeriod %}
+        {% if pageData.isRelevantPeriod or (not update.change_bo_relevant_period === "undefined") and (not update.trustee_involved_relevant_period === "undefined") and (not update.change_beneficiary_relevant_period === "undefined") %}
```

Comment:
pageData.isRelevantPeriod is already set in the controller based on the three other conditions, so both sides of your `or` are the same, I don't think we need to update any of the logic on these pages :)

I realise it is confusing the way I've done this page, I want to move to us always setting RelevantPeriod in the controller instead of checking `update.___` in the template, but at the moment we have both.